code/game/gamemodes/clock_cult/clock_mobs.dm

```diff
@@ -0,0 +1,279 @@
+					attacktext = initial(attacktext)
+				if(25 to 50) //Damage decrease, but not speed
+					melee_damage_lower = 7
+					melee_damage_upper = 7
+					attacktext = "lightly slashes"
+				if(50 to 75) //Speed decrease
+					speed = 2
+				if(75 to 99) //Massive speed decrease and weak melee attacks
+					speed = 3
+					melee_damage_lower = 5
+					melee_damage_upper = 5
+					attacktext = "weakly slashes"
+				if(99 to 100)
+					src << "<span class='userdanger'>The fatigue becomes too much!</span>"
+					if(host)
+						src << "<span class='userdanger'>You retreat to [host] - you will have to wait before being deployed again.</span>"
+						host << "<span class='userdanger'>[true_name] is too fatigued to fight - you will need to wait until they are strong enough.</span>"
+						recovering = TRUE
+						return_to_host()
+					else
+						qdel(src) //Shouldn't ever happen, but...
+
+/mob/living/simple_animal/hostile/clockwork_marauder/death(gibbed)
+	..(TRUE)
+	emerge_from_host()
+	visible_message("<span class='warning'>[src]'s equipment clatters lifelessly to the ground as the red flames within dissipate.</span>", \
+	"<span class='userdanger'>Your equipment falls away. You feel a moment of confusion before your fragile form is annihilated.</span>")
+	playsound(src, 'sound/magic/clockwork/anima_fragment_death.ogg', 100, 1)
+	new/obj/item/clockwork/component/replicant_alloy/fallen_armor(get_turf(src))
+	qdel(src)
+	return 1
+
+/mob/living/simple_animal/hostile/clockwork_marauder/Stat()
+	..()
+	if(statpanel("Status"))
+		stat(null, "Fatigue: [fatigue]/[fatigue_recall_threshold]")
+		stat(null, "Current True Name: [true_name]")
+		stat(null, "Host: [host ? host : "NONE"]")
+		stat(null, "You are [recovering ? "too weak" : "able"] to deploy!")
+
+/mob/living/simple_animal/hostile/clockwork_marauder/Hear(message, atom/movable/speaker, message_langs, raw_message, radio_freq, list/spans)
+	..()
+	if(findtext(message, true_name) && is_in_host()) //Called or revealed by hearing their true name
+		if(speaker == host)
+			emerge_from_host()
+		else
+			src << "<span class='warning'><b>You hear your true name and partially emerge before you can stop yourself!</b></span>"
+			host.visible_message("<span class='warning'>[host]'s skin flashes crimson!</span>", "<span class='warning'><b>Your marauder instinctively reacts to its true name!</b></span>")
+
+/mob/living/simple_animal/hostile/clockwork_marauder/say(message)
+	if(is_in_host())
+		src << "<span class='warning'>You cannot speak while inside of your host! Used the Linked Minds verb instead.</span>"
+		return 0
+	..()
+
+/mob/living/simple_animal/hostile/clockwork_marauder/adjustHealth(amount) //Fatigue damage
+	for(var/mob/living/L in range(3, src))
+		if(L.null_rod_check()) //Null rods allow direct damage
+			src << "<span class='userdanger'>The power of a holy artifact bypasses your armor and wounds you directly!</span>"
+			. = ..()
+			break
+			return 0
+	adjust_fatigue(amount)
+	return 1
```

Comment:
wait, better; return adjust_fatigue(amount) and adjust that proc to return the resulting fatigue taken